### PR TITLE
Add Apple Watch tutorial

### PR DESCRIPTION
**Pull request by eospi**
I created a quick tutorial demonstrating how to add an Apple Watch target to a Flutter app. With new support for bitcode in Flutter, it is now possible to create watch extensions for a Flutter app.

### CODE REVIEW

File: src/docs/development/platform-integration/apple-watch.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Adding an Apple Watch extension"
+description: "Learn how to add an Apple Watch target to a Flutter app"
+---
+
+While you cannot build an Apple Watch app with Flutter, it is possible to add a native Apple Watch extension to a Flutter app.
+
+## Step 1: Enable bitcode in Xcode
+Apple Watch targets require bitcode to be enabled, so follow the steps [here](https://github.com/flutter/flutter/wiki/Creating-an-iOS-Bitcode-enabled-app-(experimental)) to use bitcode in your app.
```
Comment (sfshaza2) [inline src/docs/development/platform-integration/apple-watch.md]: Hi, @eospi! Thanks for your PR! 

Linking on "here" is bad form. Can you rephrase to:
Apple Watch targets require bitcode to be enabled, so follow the steps in [Creating an iOS Bitcode enabled app][]
to use bitcode in your app.

[Creating an iOS Bitcode enabled app]: LINK (The link in your text doesn't work for me. Can you double check?)

File: src/docs/development/platform-integration/apple-watch.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+## Step 2: Add an Apple Watch target
+In the menu, select File > New > Target. Once the dialog opens, select *watchOS* at the top and click *Watch App for iOS App*. Click *Next*, enter a product name, and select *Enter*.
```
Comment (sfshaza2) [inline src/docs/development/platform-integration/apple-watch.md]: We use bold for menu items ( https://developers.google.com/style/ui-elements ) =>
In the menu, select **File > New > Target**. Once the dialog opens,
select **watchOS** at the top and click **Watch App for iOS App**.
Click **Next**, enter a product name, and select **Enter**.

(Also, your line length is too long. We generally break on phrases and use no more than 80 chars.)

**Comment by sfshaza2** [comment]
Have we lost you, @eospi?

**Comment by eospi** [comment]
Thanks for looking over the PR! I finished midterms yesterday. I've just made the changes.
